Foody/Onboarding/Login/LoginViewController.swift

```diff
@@ -6,29 +6,112 @@
 //
 
 import UIKit
+import FirebaseAuth
+import Firebase
 
 class LoginViewController: UIViewController {
 
+    @IBOutlet weak var loginImageView: UIImageView!
+    
+    @IBOutlet weak var loginLbl: UILabel!
+    @IBOutlet weak var welcomeLbl: UILabel!
+    
+    @IBOutlet weak var rememberMeSwitch: UISwitch!
+    
+    @IBOutlet weak var emailTextFld: UITextField!
+    @IBOutlet weak var passwordTextFld: UITextField!
+    
+    @IBOutlet weak var loginBttn: UIButton!
+    @IBOutlet weak var registerBttn: UIButton!
+    
```

Comment:
let defaults: UserDefaults = UserDefaults.standard. What do you think if we initilaze deafults here and use everywhere. No need to create everytime.  It is code duplication and we don't want duplication.

Reply:
OK. I got it. Thank you for your comment.